### PR TITLE
HAWQ-991. Check bucketnum and encoding consistency in hawq register.

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @ictmalili @wengyanqing

**Comment by xunzhang** [comment]
also cc @wcl14

File: tools/bin/hawqregister [hunk 1]
```diff
@@ -242,6 +253,13 @@ class HawqRegister(object):
                     logger.error('Files to be registered must be multiple times to the bucket number of hash table.')
                     sys.exit(1)
 
+        def check_database_encoding():
+            encoding_indx = self.accessor.get_database_encoding_indx(self.database)
+            encoding = self.accessor.get_database_encoding(encoding_indx)
+            if self.encoding.strip() != encoding:
+                logger.error('Database encoding is not consistent.')
```
Comment (ictmalili) [inline tools/bin/hawqregister]: Shall we print out the two encodings?

Comment (xunzhang) [inline tools/bin/hawqregister]: ok

Comment (xunzhang) [inline tools/bin/hawqregister]: done

**Comment by xunzhang** [comment]
Merged into master.

**Pull request closed without merging**